content/en/docs/single-repo/overview.md

```diff
@@ -12,4 +12,4 @@ weight: 31
 
 Demo for git workflow on a single repository:
 
-{{< figure src="/images/git-repo-single.gif" caption="Fig: git-repo for multiple repositories" >}}
+{{< figure src="/images/git-repo-single.gif" caption="Fig: git-repo for single repositories" >}}
```

Comment:
Will merge with this update:

    single repositories => a single repository

Reply:
Fixed.